.github/workflows/build.yml

```diff
@@ -50,9 +51,9 @@ jobs:
         base_image_flavor: [main, asus, framework, surface]
         base_name: [bazzite, bazzite-deck, bazzite-nvidia]
         base_image_name: [kinoite, silverblue]
-        major_version: [39]
+        major_version: [40]
```

Comment:
Why not ship 39 and 40 concurrently?

Reply:
> Why not ship 39 and 40 concurrently?

With the amount of package we have to manage it's a huge effort to maintain both, and gaming with older packages/Mesa is a bit silly.